share/completions/jj.fish

```diff
@@ -1 +1,10 @@
-jj util completion fish | source
+# The reason for `__this-command-does-not-exist` is that, if dynamic completion
+# is not implemented, we'd like to get an error reliably. However, the
+# behavior of `jj` without arguments depends on the value of a config, see
+# https://jj-vcs.github.io/jj/latest/config/#default-command
+if set -l completion (COMPLETE=fish jj __this-command-does-not-exist 2>/dev/null)
+    # jj is new enough for dynamic completions to be implemented
+    echo "$completion" | source
+else
+    jj completion fish | source
```

Comment:
changing from `jj util completion fish` to `jj completion fish` (which is no longer suported in current `jj`) seems odd.
I guess you know that this works for all relevant versions?

Reply:
No, this is a silly mistake. Thanks for catching it!